### DISCUSSION TITLE
How can we get different expressions of pi creatures?

### DISCUSSION THREAD

**Discussion by CaftBotti** [ManimCommunity/manim#2947] [Ideas]
Hi everyone. I saw many cute pi creatures in 3b1b videos. Then i tried to make picreature animations. The only problem is that we only have 1 expression of them(plain) so that's boring. I really really hope to have more pi creatures, thanks you a lot! Welcome to discuss.

**Comment by SuperUser-Vermeylen** [comment]
The pi-creatures are probably more of proprietary part of Manim than the library.
The reason the community version was forked stemmed from his original code being only his and having all the quirks of personal code.
I'm not sure the library needs specific art to come with it, unless an artist wants to donate some unique creatures to the community version.

**Comment by MrDiver** [comment]
You can probably hack around a little bit with SVGMobject and import a few pi creatures and group them together in a custom mobject such that they move all in sinc and then use a transform animation on the mouth and the eyes with an AnimationGroup should give you a good expression Transformation. I currently don't have time to experiment with SVG Pi creatures but i think many people would enjoy a tutorial on custom creatures in Manim!

**Reply by CaftBotti** [reply]
Thanks so much. Maybe 3b1b reserved © of picreature.

**Comment by CaftBotti** [comment]
See [here](https://github.com/CaftBotti/manim_pi_creatures)
